### PR TITLE
Release/2.1.0

### PR DESCRIPTION
**Pull request by vigi86**
Added `Artificial Intelligence & Machine Learning` course modules

I wasn't able to update Firefox and Chrome extentions.

### CODE REVIEW

File: tools/parse_module_type.py [hunk 1]
```diff
@@ -123,6 +124,31 @@ def parseWebsite():
...
+            module_type_title = section.get_element_by_id(
+                module_type_html_id, None)
+            if module_type_title is not None:
+                modules = section.find_class(
+                    'columns col-collapse small-12 print-12 download-text')
+                for module in modules:
+                    module_name = str(etree.tostring(module))
+                    #print(module_name)
+                    if '(Angewandte)' in module_name:
+                        # this is needed due to the module "(Angewandte) Mathematik 2 (MAT2) C12/C16"
```
Comment (vigi86) [inline tools/parse_module_type.py:144]: Yeah, I just copy-pasted... 😒

File: src/manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "HSLU simple MEP results",
-  "version": "2.0.15",
+  "version": "2.1.0",
```
Comment (eddex) [inline src/manifest.json:4]: fräch!

**Review by eddex — APPROVED** [review]
next time don't just update the version, this is our job :)

File: tools/parse_module_type.py [hunk 3]
```diff
@@ -224,6 +252,15 @@ def prequisitesCheck():
         return False
     return True
```
Comment (Lextum) [inline tools/parse_module_type.py:248]: This return is too much, remove please.

File: tools/parse_module_type.py [hunk 4]
```diff
@@ -224,6 +252,15 @@ def prequisitesCheck():
...
 
+    f = Path('./modulbeschriebe_ai.html')
+    if not f.is_file():
+        print('ERROR: file \'./modulbeschriebe_ai.html\' does not exist.')
+        print('To get started download the html file from \
+https://mycampus.hslu.ch/de-ch/info-i/dokumente-fuers-studium/bachelor/moduleinschreibung/modulbeschriebe/bachelor-artificial-intelligence-machine-learning/ \
+and save it as \'tools/modulbeschriebe_wi.html\'.')
```
Comment (Lextum) [inline tools/parse_module_type.py:255]: modulbeschriebe_wi.html should be modulbeschriebe_ai.html

**Review by Lextum — CHANGES_REQUESTED** [review]

File: tools/parse_module_type.py (hunk 3 above)
Comment (vigi86) [reply tools/parse_module_type.py:248]: The `return True`? That's your code. I can do, but you sure? If so, should I also delete line 262 then?

File: src/manifest.json (hunk 2 above)
Comment (vigi86) [reply src/manifest.json:4]: Just followed the README-Development-Create a new release... 😅  tued mer sorry, hehe

File: tools/parse_module_type.py (hunk 4 above)
Comment (vigi86) [reply tools/parse_module_type.py:255]: Duh... 🤦🏻‍♂️

**Comment by vigi86** [comment]
**Wait with merging!**

I found that the AI acronym is missing in the main.js, so this will probably blow up and cause an exception. 

I asked if the German and English title is the same. I guess it's _bachelor of science in artificial intelligence & machine learning_ in both. But I'm waiting a response from the secratariat, because _"die zueständig Person esch grad need erreichbar"_

**Comment by vigi86** [comment]
Found out that *Modulbeschriebe* for AI are only available in German, so I'd say we good to go.

**Review by Lextum — APPROVED** [review]
perfect, looks fine.

**Pull request merged**